tests/integration/testdata/sync/039-avoid-entity-rewrite-with-id-on-select-tag-mismatch/acl-group-initial.yaml

```diff
@@ -0,0 +1,26 @@
+_format_version: "3.0"
+consumers:
+- acls:
+  - group: consumer's ACL
+    id: f2be545c-45c5-46e2-8acf-00ff2d899073
+    tags:
+    - before
+  custom_id: custom_id_1
+  id: 27d847fb-7aeb-4cb7-adb5-eccdd0ff1d42
+  username: consumer1
+  tags:
+  - before
+plugins:
```

Comment:
Do we require plugin config for this test? We aren't testing for acl functionality anyway.
Does anything error out if it is not present?

Reply:
Same for other cred types.

Reply:
Yep, plugin was not needed - i think I missed it because it was needed on UI